### PR TITLE
Http doc fixes [WIP]

### PR DESCRIPTION
**Pull request by dprats**
Fixes to the documentation for using HTTPS outcalls.

Status:

:white_check_mark: motoko sample code of GET
:white_check_mark: motoko sample code of POST
:white_check_mark: rust sample code of GET
:white_check_mark: rust sample code of POST

### CODE REVIEW

File: docs/developer-docs/integrations/https-outcalls/https-outcalls-get.md [hunk 1]
```diff
@@ -406,13 +477,24 @@ async fn get_icp_usd_exchange() -> String {
...
+    };
+
     //note "CanisterHttpRequestArgument" and "HttpMethod" are declared in line 4
     let request = CanisterHttpRequestArgument {
         url: url.to_string(),
         method: HttpMethod::GET,
         body: None,               //optional for request
         max_response_bytes: None, //optional for request
-        transform: None,          //optional for request
+        // transform: None,          //optional for request
```
Comment (yotamhc) [inline docs/developer-docs/integrations/https-outcalls/https-outcalls-get.md:496]: Maybe remove this line?

File: docs/developer-docs/integrations/https-outcalls/https-outcalls-post.md [hunk 2]
```diff
@@ -164,7 +196,8 @@ actor {
         //note: type of `body` is ?[Nat8] so we pass it here as "?request_body_as_nat8" instead of "request_body_as_nat8"
         body = ?request_body_as_nat8; 
         method = #post;
-        transform = null; //optional for request
+        transform = ?transform_context;
+        // transform = null; //optional for request
```
Comment (yotamhc) [inline docs/developer-docs/integrations/https-outcalls/https-outcalls-post.md:200]: Same here

**Review by yotamhc — APPROVED** [review]

File: docs/developer-docs/integrations/https-outcalls/https-outcalls-get.md [hunk 3]
```diff
@@ -275,17 +305,29 @@ module Types {
         context : Blob;
     };
 
-    //2.2 This type describes the arguments the transform function needs
+    //2.2 These type describes the arguments the transform function needs
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/https-outcalls/https-outcalls-get.md:308]:
```suggestion
    //2.2 These types describes the arguments the transform function needs
```

File: docs/developer-docs/integrations/https-outcalls/https-outcalls-get.md [hunk 4]
```diff
@@ -497,20 +652,38 @@ If successful, the terminal should return canister URLs you can open:
...
+```
+
+You can see play with the dapp's `get_icp_usd_exchange` method on-chain here: [https://a4gq6-oaaaa-aaaab-qaa4q-cai.raw.ic0.app/?id=fm664-jyaaa-aaaap-qbomq-cai](https://a4gq6-oaaaa-aaaab-qaa4q-cai.raw.ic0.app/?id=fm664-jyaaa-aaaap-qbomq-cai).
 
 ## Additional resources
 
-* Sample code of [HTTP GET requests in Rust](https://github.com/dfinity/examples/tree/master/rust/send_http_get)
-* Sample code of [HTTP GET requests in Motoko](https://github.com/dfinity/examples/tree/master/motoko/send_http_get)
\ No newline at end of file
+- Sample code of [HTTP GET requests in Rust](https://github.com/dfinity/examples/tree/master/rust/send_http_get)
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/https-outcalls/https-outcalls-get.md:688]:
```suggestion
- Sample code of [HTTP GET requests in Rust.](https://github.com/dfinity/examples/tree/master/rust/send_http_get)
```

File: docs/developer-docs/integrations/https-outcalls/https-outcalls-get.md [hunk 5]
```diff
@@ -497,20 +652,38 @@ If successful, the terminal should return canister URLs you can open:
...
+- Sample code of [HTTP GET requests in Motoko](https://github.com/dfinity/examples/tree/master/motoko/send_http_get)
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/https-outcalls/https-outcalls-get.md:689]:
```suggestion
- Sample code of [HTTP GET requests in Motoko.](https://github.com/dfinity/examples/tree/master/motoko/send_http_get)
```

File: docs/developer-docs/integrations/https-outcalls/https-outcalls-post.md [hunk 6]
```diff
@@ -9,6 +10,10 @@ The sample code is in both Motoko and Rust. This sample canister sends a `POST`
 
 This example takes less than 5 minutes to complete.
 
+:::caution
+The HTTPS Outcalls feature only works for sending HTTP POST requests to servers or API endpoints that support **IPV6**.
```
Comment (jessiemongeon1) [inline docs/developer-docs/integrations/https-outcalls/https-outcalls-post.md:14]:
```suggestion
The HTTPS outcalls feature only works for sending HTTP POST requests to servers or API endpoints that support **IPV6**.
```

**Review by jessiemongeon1 — APPROVED** [review]

File: docs/developer-docs/integrations/https-outcalls/https-outcalls-get.md (hunk 1 above)
Comment (dprats) [reply docs/developer-docs/integrations/https-outcalls/https-outcalls-get.md:496]: will do

**Pull request merged**